linkerd/app/outbound/src/endpoint.rs

```diff
@@ -53,6 +53,12 @@ pub struct TcpLogical {
     pub profile: Option<profiles::Receiver>,
 }
 
+#[derive(Clone, Debug)]
```

Comment:
nit: while working on the test changes i realized that it might not be great to derive `Debug` for `TcpLogical`, since it has a MPSC receiver in it, which dumps a bunch of internal state that we don't care about...we might want a custom `Debug` impl for `TcpLogical` because of that.

not a blocker, can fix it later; just something i remembered while looking at this.